prov/rxm/src/rxm_domain.c

```diff
@@ -893,6 +910,9 @@ int rxm_domain_open(struct fid_fabric *fabric, struct fi_info *info,
 					 FI_PEER, &peer_context);
 			if (ret)
 				goto err5;
+
+			rxm_domain->offload_coll_mask |=
+			get_capabilities_mask(rxm_domain->offload_coll_domain);
```

Comment:
nit: indent function call at least an extra tab.

Reply:
Done with 4 spaces to keep 80 characters limit.